src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -114,7 +117,30 @@ public void setUpdateDate(Timestamp updateDate) {
     this.updateDate = updateDate;
   }
 
-  /**
+  public Integer getCountUsers() {
+    List<String> emails = null;
+    if (Objects.nonNull(data)) {
+      ArrayList<Collaborator> collabs = new ArrayList<>();
+      if (Objects.nonNull(data.getInternalCollaborators())) {
+        collabs.addAll(data.getInternalCollaborators());
+      }
+      if (Objects.nonNull(data.getLabCollaborators())) {
+        collabs.addAll(data.getLabCollaborators());
+      }
+      emails = collabs.stream().map(collaborator -> Objects.requireNonNull(collaborator).getEmail()).collect(Collectors.toList());
+
+      if (Objects.nonNull(data.getAcademicEmail())) {
+        emails.add(data.getAcademicEmail());
+      }
+      if (Objects.nonNull(data.getPiEmail())) {
+        emails.add(data.getPiEmail());
+      }
```

Comment:
We don't want to get `User` information from the data access request, we want to get it from the `User`'s `ResearcherProperty` values. In the ticket, I suggested that we pull this feature out into a separate ticket so we can address the design in a better way. Let's revert this code and make sure we're doing the right thing in the metrics class, which we're not doing now. For example, we need the `User` which we don't have in the constructor right now.